Makefile

```diff
@@ -27,6 +30,12 @@ test:
 	xcodebuild test \
 		-scheme ComposableCoreMotion \
 		-destination platform="$(PLATFORM_IOS)"
+	xcodebuild test \
+		-scheme ComposableCoreMotion \
+		-destination platform="$(PLATFORM_MACOS)"
+	xcodebuild test \
+		-scheme ComposableCoreMotion \
+		-destination platform="$(PLATFORM_TVOS)"
```

Comment:
As long as we support building core motion for any target we should probably also run the tests.